runtime/battery-station/src/parameters.rs

```diff
@@ -74,6 +75,26 @@ parameter_types! {
     /// the juror has to pay.
     pub const StakeWeight: u128 = 2 * BASE;
 
+    // Global Disputes
+    /// Pallet identifier
+    pub const GlobalDisputesPalletId: PalletId = GLOBAL_DISPUTES_PALLET_ID;
+    /// Vote lock identifier, mainly used for the LockableCurrency on the native token.
+    pub const VoteLockIdentifier: LockIdentifier = *b"zge/vote";
+    /// The maximum number of owners for a voting outcome for private API calls of `push_voting_outcome`.
+    pub const MaxOwners: u32 = 10;
+    /// The maximum number of market ids (participate in multiple different global disputes at the same time) for one account to vote on outcomes.
+    pub const MaxGlobalDisputeVotes: u32 = 50;
```

Comment:
`MaxGlobalDisputeVotes` is necessary, because each lock is stored on-chain and we iterate through each global dispute in [this `retain`](https://github.com/zeitgeistpm/zeitgeist/blob/5bdf9b461a20d21b0421921c00822e51737d29df/zrml/global-disputes/src/lib.rs#L321-L323). The locked funds are not in risk. Voting uses a `LockableCurrency`, which can't be transferred or slashed.
The setting means, that a user can only lock voting funds for `50` different global disputes. When the user unlocks with `unlock_vote_balance`, the user can participate in 50 global disputes again.